libs/Options/Customizer.php

```diff
@@ -880,16 +880,45 @@ public function customizer_style_overrides(){
 					#bigheadingcon {background: <?php echo get_theme_mod('nebula_primary_color'); ?>;}
 				<?php endif; ?>
 
+				<?php if ( get_theme_mod('nebula_hero_overlay_color') || get_theme_mod('nebula_hero_overlay_opacity') ): ?>
+					<?php $hero_overlay = ( get_theme_mod('nebula_hero_overlay_color') )? 'background: ' . get_theme_mod('nebula_hero_overlay_color') . ' ;' : ''; ?>
+					<?php $hero_overlay .= ( !is_null(get_theme_mod('nebula_hero_overlay_opacity')) )? 'opacity: ' . get_theme_mod('nebula_hero_overlay_opacity') . ';' : ''; ?>
+					<?php $hero_overlay .= 'animation: none;'; ?>
```

Comment:
Since this is no longer an inline style, we don't need to concatenate these anymore- let's update this to something a little more readable. Either each property could be a variable, or we could set a single selector multiple times with different properties.